packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.tsx

```diff
@@ -162,12 +163,7 @@ export function EnumFacetFilter(props: EnumFacetFilterProps) {
       hideCollapsible={hideCollapsible}
       onHoverOverValue={() => {
         // SWC-6698: delay the query execution (via the debounce) when an item is hovered over
-        executeQueryRequest(
-          request => {
-            return cloneDeep(request)
-          },
-          { debounce: true },
-        )
```

Comment:
Calling `executeQueryRequest` would recreate the query object and trigger unnecessary rerenders every time an option was hovered.

Replace this call a function that does the bare minimum to avoid unnecessarily rerendering most of the QueryWrapper tree.